samples/grpc/shopping-analytics-service-scala/kubernetes/shopping-analytics-service-cr.yml

```diff
@@ -0,0 +1,22 @@
+apiVersion: "v1"
+kind: "Namespace"
+metadata:
+  name: "shopping"
+---
+apiVersion: akka.lightbend.com/v1
+kind: AkkaMicroservice
```

Comment:
Let's include a deployment.yaml instead of the CR for the Platform operator. See the Java samples.

Reply:
Should we even include the k8 stuff in these samples at all?

Reply:
I dropped the k8 descriptors completely instead, better if we keep those to the platform guide and don't need to maintain those all over the place in samples IMO

Reply:
ok